templates/CRM/Pledge/Form/Pledge.tpl

```diff
@@ -144,6 +149,13 @@
 // bind first click of accordion header to load crm-accordion-body with snippet
 // everything else taken care of by cj().crm-accordions()
 cj(document).ready( function() {
+
+  cj('#currency').closest('tr').next('tr').find('.currency-symbol').text(cj('#currency option:selected').text());
+  // if there are more than one currency enabled.
+   cj('#currency').change(function(){
+     cj('#currency').closest('tr').next('tr').find('.currency-symbol').text(cj(this).find(':selected').text());
```

Comment:
I think a display of whole text (`USD ($) 100`) looks little unusual than only symbols (`$ 100`). IMO, it should only display a symbol before the amount on the form. @colemanw Any suggestions ?

Also, instead of navigating to the `currency-symbol` class through different methods, I think we should directly select the element from the starting point as we don't have the class name used anywhere else in the form. Something like -

    cj('.currency-symbol').text(...);